src/Umbraco.Web.UI.Client/src/views/directives/umb-sections.html

```diff
@@ -16,7 +16,7 @@
                 </a>
             </li>
             
-            <li class="expand" ng-show="needTray">
+            <li class="expand" ng-class="{ 'open': showTray === true }" ng-show="needTray">
                 <a href ng-click="trayClick()">
                     <i class="icon icon-arrow-right"></i>
                 </a>
```

Comment:
If you prefer switch off arrow-right icon the arrow-left instead of rotating, you could insert this instead
```<i class="icon" ng-class="{ 'icon-arrow-left': showTray === true, 'icon-arrow-right': showTray === false }"></i>```

and then these lines https://github.com/bjarnef/Umbraco-CMS/blob/dev-v7-U4-4507/src/Umbraco.Web.UI.Client/src/less/sections.less#L120-L136 is not necessary and it just switch between right and left arrow without animation. But I like the solution I have committed here :)